src/main/webapp/quizFunctions.js

```diff
@@ -13,51 +13,31 @@ if(document.getElementById("sound").onclick = function() {
         status_of_music = "pause";
     }
 });
-
-const quizQuestions = [
-    "Which plant has the most food growing from it?",
-    "Which plant has the prettiest colors?",
-    "Which plant is likely to grow the fastest?",
-    "Which plant is likely to require the most maintenance to take care of?",
-    "Which plant is most likely to be poisonous?",
-    "Which plant is most likely to survive really dry weather?",
-    "Which plant is the largest?",
-    "Which plant looks edible?",
-    "Which plant looks the healthiest?",
-    "Which plant looks the most serene?",
-    "Which plant needs the most sunlight?",
-    "Which plant needs the most water?",
-    "Which plant represents you?",
-    "Which plant represents your best friend?",
-    "Which plant will most likely impress your friends and family?",
-    "Which plant would look the best inside as a houseplant?",
-    "Which plant would look the best outside in a garden?",
-    "Which plant would you give as a gift?"
-];
-let question_to_be_answered = document.createElement('h5');
-
-function generateQuizQuestions() {
-    let question =  quizQuestions[Math.floor(Math.random() * quizQuestions.length)];
-    question_to_be_answered.innerText = question;
-    question_to_be_answered.id = "questions";
-    document.getElementById('quiz_question').appendChild(question_to_be_answered);
-}
-
+ 
 document.getElementById("start_quiz").onclick = function() {
-    document.getElementById("quiz_time").style.display = "block";
-    document.getElementById("start_quiz").style.display = "none";
-    generateQuizQuestions();
-}
+    fetch("/user-quiz-status-servlet").then(response => response.json()).then((tasks) => {
+        if(tasks == true){
+            console.log("You already took this quiz!");
```

Comment:
can u display it on the page instead of console.logging this?

Reply:
Yes, I added an alert here!